src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const { slug = '' } = useParams();
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
it's not clear why this variable has `true` value by default
we can update logic in future, and forget to change the initial value for this variable

let's define it as `true` only before making some request for loading data